src/Microsoft.Framework.DependencyInjection.Interfaces/project.json

```diff
@@ -14,6 +14,7 @@
                 "System.Diagnostics.Debug": "4.0.10-beta-*",
                 "System.Globalization": "4.0.10-beta-*",
                 "System.Linq": "4.0.0-beta-*",
+                "System.Linq.Expressions": "4.0.10-beta-*",
```

Comment:
Could you remove this dependency from `Microsoft.Framework.DependencyInjection\project.json` if it exists?